.github/ISSUE_TEMPLATE/3_Badge_request.yml

```diff
@@ -0,0 +1,55 @@
+name: '💡 Badge Request'
+description: Ideas for new badges
+labels: ['service-badge']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Ideas for new badges
+
+        This issue template is for suggesting new badges which **fetch and display data from an upstream service**. If your suggestion is for a static badge (which shows the same information every time it is requested), it is [already possible to make these](https://github.com/badges/shields/blob/master/doc/static-badges.md). We don't add specific routes for badges which only show static information.
```

Comment:
`static-badges.md` is added in this PR so the link will be dead till we merge it

Reply:
To be coherent with the other long lines, let's break down this line into shorter lines that fit within the screen.

Reply:
Just to explain 9bd932a

The reason I originally wrote this on a single line is because what we are actually writing at this point is Github-Flavoured markdown embedded inside a multi-line yaml string (which then generates an issue form :smile: ). If you write

```yml
value: |
    A line
    Another line
```

that's going to render as 2 paragraphs, and I wanted this whole thing to render as one paragraph. The other makrdown strings in this file are bullet lists, so they split across multiple lines naturally.

I think the solution to this is to switch from a _literal_ multi-line yaml string `|` to a _folded_ multi-line yaml string `>` (see https://yaml-multiline.info/ ) which should allow us to write this paragraph on multiple lines in the yaml but then have it render as a single paragraph when someone raises an issue. I'm going to have to double check this once we merge it though and might need to tweak it. I'm more concerned about the user seeing the right thing in the issue form than the how the raw yaml looks.

Reply:
Cool, thanks, I didn't know about the distinction between a literal multi-line YAML string (`|`) and a folded multi-line YAML string (`>`). 
And you're right that it is more important how the result will look for the user vs what the raw YAML will look like :)